boilerplate/flyte/golang_test_targets/Makefile

```diff
@@ -10,7 +10,7 @@ download_tooling: #download dependencies (including test deps) for the package
 
 .PHONY: lint
 lint: download_tooling #lints the package for common code smells
-	GL_DEBUG=linters_output,env golangci-lint run --deadline=5m --exclude deprecated -v
+	GL_DEBUG=linters_output,env golangci-lint run --deadline=7m --exclude deprecated -v
```

Comment:
this needs an update in boilerplate and it will automatically percolate. 
But do you need this?

Reply:
I did that because linting was timing out. I think it's because of added dependencies. Let me try again.